lang/de/auth_failed.txt

```diff
@@ -0,0 +1,5 @@
+Achtung!
+
+Der konfigurierte Schwellwert für fehlgeschlagene Anmeldeveruche wurde überschritten!
+
+Sie sollen sich an den Administrator wenden, um das Problem zu untersuchen.
```

Comment:
Remove the last line. Instead add info on how to assess the situation. Eg. by logging into the wiki and checking the logs

Reply:
Done.